protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -36,7 +36,7 @@
 from starkware.starknet.business_logic.state.state import StateSyncifier
 from protostar.profiler.pprof import serialize, to_protobuf
 from protostar.profiler.contract_profiler import RuntimeProfile, build_profile
-from protostar.profiler.profile import (
+from protostar.profiler.transaction_profiler import (
```

Comment:
Imports could be relative.

Reply:
https://github.com/software-mansion/protostar/issues/978